app/services/analytics.rb

```diff
@@ -34,19 +34,16 @@ def uuid
   EMAIL_CHANGE_REQUEST = 'Email Change Request'.freeze
   EMAIL_CONFIRMATION = 'Email Confirmation'.freeze
   IDV_BASIC_INFO_VISIT = 'IdV: basic info visited'.freeze
-  IDV_FAILED = 'IdV: failed'.freeze
+  IDV = 'IdV'.freeze
   IDV_FINANCE_VISIT = 'IdV: finance visited'.freeze
   IDV_INTRO_VISIT = 'IdV: intro visited'.freeze
   IDV_PHONE_RECORD_VISIT = 'IdV: phone of record visited'.freeze
   IDV_REVIEW_VISIT = 'IdV: review info visited'.freeze
-  IDV_SUCCESSFUL = 'IdV: successful'.freeze
   INVALID_AUTHENTICITY_TOKEN = 'Invalid Authenticity Token'.freeze
-  INVALID_SERVICE_PROVIDER = 'Invalid Service Provider'.freeze
   OTP_DELIVERY_SELECTION = 'OTP: Delivery Selection'.freeze
   MULTI_FACTOR_AUTH = 'Multi-Factor Authentication'.freeze
   MULTI_FACTOR_AUTH_ENTER_OTP_VISIT = 'Multi-Factor Authentication: enter OTP visited'.freeze
   MULTI_FACTOR_AUTH_MAX_ATTEMPTS = 'Multi-Factor Authentication: max attempts reached'.freeze
-  PAGE_NOT_FOUND = 'Page Not Found'.freeze
```

Comment:
Might consider putting these "other cleanup" bits in a separate commit or even separate PR to keep refactor and cleanup separate in git history